src/FourierTransforms.jl/ctfft.jl

```diff
@@ -374,13 +383,17 @@ const CTComplex = Union{Complex{Float32},Complex{Float64}} # types of pregenerat
 # include hard-coded transcendental constants.  The following kernels
 # are generic to any floating-point type.
 const generic_kernel_sizes = Set([1,2,4])
-for n in (1,2,4)
-    @eval @nontwiddle($(Complex{<:AbstractFloat}), true, $n)
-    @eval @nontwiddle($(Complex{<:AbstractFloat}), false, $n)
-end
-for forward in (true,false)
-    @eval @twiddle($(Complex{<:AbstractFloat}), $forward, 2)
-    @eval @twiddle($(Complex{<:AbstractFloat}), $forward, 4)
+for T in [AbstractFloat, Interval]
+    # This loop is kind of bad, because it creates the code twice for
+    # both abstract types respectively ... that's not what we want, but it works ...
+    for n in (1,2,4)
+        @eval @nontwiddle($(Complex{<:T}), true, $n)
+        @eval @nontwiddle($(Complex{<:T}), false, $n)
+    end
```

Comment:
I don't understand why this is needed?

Reply:
Not sure exactly, what you are referring to.